checks.nix

```diff
@@ -0,0 +1,169 @@
+{
+  system,
+  packages,
+  pkgs,
+}:
+
+let
+  phpVersions = [
```

Comment:
How about passing all the versions of PHP as packages from the `flake.nix` ?

We can then avoid maintaining 2 lists of PHP versions.

Reply:
Thanks, wanted to do that but forgot about it.